mayhem/fuzz-pywhat.py

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/python3
```

Comment:
One more thing as I'm looking at this - I don't think using `sys.maxsize` is a good idea here, as there's no guarantee on the length of the fuzzed data (could be smaller). Can you instead use `fdp.remainingBytes()` here?